StandardRecord/StandardRecord.h

```diff
@@ -45,11 +47,23 @@ namespace caf
     double RecoLepEnNumu;
     double RecoHadEnNumu;
 
+    double HadEVisReco_ND;
```

Comment:
All these variables are going to need to go in the duneanaobj version of StandardRecord.

Reply:
Can you make a new PR for that?